rllib/utils/replay_buffers/simple_replay_buffer.py

```diff
@@ -32,6 +32,8 @@ def add(self, batch: SampleBatchType, **kwargs) -> None:
     @DeveloperAPI
     @override(ReplayBuffer)
     def sample(self, num_items: int, **kwargs) -> SampleBatchType:
+        if len(self.replay_batches) == 0:
+            raise ValueError("Trying to sample from an empty buffer.")
         return random.choice(self.replay_batches)
```

Comment:
Sorry, I know, this probably doesn't concern this PR, should we respect num_items > 1 here? Currently, this arg is completely ignored and the user doesn't get a warning about this.